Slightly Higher Chances

func-octo opened the pull request:
Gist:
nz_zombie_special_burning should have a slightly higher chance to spawn.

Proposed File Changes:
- Increased chance of burning zombies spawning by 5% _on round 13_ in order to descrease sparsity a small bit
- Chance of normal zombies spawning decreased 5% _on round 13_ to keep math working

Commit:
Slightly Higher Chances

nz_zombie_special_burning should have a slightly higher chance to spawn

Proposed File Changes:
- Increased chance of burning zombies spawning by 5% in order to descrease sparsity a small bit

Zet0rz commented:
Remember that these config settings make them only appear on round 13, and round 23 and all rounds after that. A 25% for all rounds after round 23 is pretty high considering the already high zombie count. I can see round 13, but I'd suggest not changing 23-onward settings.

func-octo commented:
File has been changed. Only round 13 is affected now.

Commit:
Update sh_constructor.lua

Commit:
Fixed a math error

Because 80% + 10% != 100%

func-octo commented:
Has this gone stale?

Zet0rz commented:
![](http://i.imgur.com/eo2ejsJ.png)
You haven't changed the last one back?

func-octo commented:
I did. It is in commit fe30216

func-octo commented:
First I changed both, then I changed round 23 back to normal, then I fixed math because 10 + 80 doesn't equal 100 (In round 23. The chances are 90% and 10% like they were before.)

func-octo commented:
Bump (@Zet0rz See last comment for info)
